gometalinter.json

```diff
@@ -19,7 +19,6 @@
     "interfacer",
     "lll",
     "vet",
-    "gas",
```

Comment:
Do we want to remove this?

Reply:
nice catch meant to replace with gosec (which I have now done)